core/src/Kokkos_CopyViews.hpp

```diff
@@ -1311,8 +1311,8 @@ contiguous_fill_or_memset(
 template <typename ExecutionSpace, class DT, class... DP>
 inline std::enable_if_t<!(
     std::is_trivial<typename ViewTraits<DT, DP...>::const_value_type>::value &&
-    std::is_trivially_copy_assignable<
-        typename ViewTraits<DT, DP...>::const_value_type>::value)>
+    std::is_trivially_copy_assignable<std::remove_const_t<
+        typename ViewTraits<DT, DP...>::const_value_type>>::value)>
```

Comment:
```suggestion
    std::is_trivially_copy_assignable<
        typename ViewTraits<DT, DP...>::value_type>::value)>
```

Reply:
For `const` Views `value_type` is still `const` but of course `const` Views can also not be assigned. Do you still want me to change?

Reply:
How are you going to assign to it if it is `const`?  :p

Reply:
I'll just take that as a "yes, please change".

Reply:
I had read only half your comment above.  But yea please do.